synapse/models/files.py

```diff
@@ -556,6 +577,54 @@ def getModelDefs(self):
                     ('base:ext', ('str', {}), {'ro': True,
                         'doc': 'The file extension.'}),
                 )),
+
+                ('file:mime:macho:loadcmd', {}, (
+                    ('file', ('file:bytes', {}), {
+                        'doc': 'The Mach-O file containing the load command.'}),
+                    ('type', ('int', {'enums': s_l_macho.getLoadCmdTypes()}), {
+                        'doc': 'The type of the load command'}),
+                    ('size', ('int', {}), {
+                        'doc': 'The size of the load command structure in bytes.'}),
+                )),
+                ('file:mime:macho:version', {}, (
+                    ('loadcmd', ('file:mime:macho:loadcmd', {}), {
+                        'doc': 'The associated load command node.'}),
+                    ('version', ('str', {}), {
+                        'doc': 'The version of the Mach-O file encoded in an LC_VERSION load command'})
+                )),
+                ('file:mime:macho:uuid', {}, (
+                    ('loadcmd', ('file:mime:macho:loadcmd', {}), {
+                        'doc': 'The actual load command '}),
+                    ('uuid', ('str', {}), {
```

Comment:
Are there any case concerns here?

Reply:
Uhh, once we get to this point, not really. The closest I could get to a "formal" Mach-O spec states that the UUID command:

`Specifies the 128-bit universally unique identifier (UUID) for an image or for its corresponding dSYM file.`

So if we're just hex-ifying some bytes, it shouldn't be a problem. So I'll add the lower specifier to this.